app/components/preprint-status-banner/template.hbs

```diff
@@ -0,0 +1,42 @@
+<div id="preprint-status" class="container p-md">
+    <div class="flex-container">
+        {{#if model.provider.isPending}}
+            Loading...
+        {{else}}
+            <div class="status-explanation">
+                <i class="fa {{icon}} status-icon" aria-hidden="true"></i>
+                <strong>{{t status}}:</strong>
+                <span>
+                    {{t baseMessage name=(if theme.isProvider theme.provider.name (t "global.brand_name")) reviewsWorkflow=(t workflow)}} {{t bannerContent}}.
```

Comment:
Matter of taste, but it might be cleaner and more readable to put this in a computed property that returns the full translated string. Could do that for all the other translated strings too... not sure I have a preference.